src/lib/containers/download_list_v2/CreatePackageV2.tsx

```diff
@@ -50,7 +55,12 @@ export const CreatePackageV2 = (props: CreatePackageV2Props) => {
     }
     setAlert({ message: '', variant: undefined, className: undefined })
     setIsLoading(true)
+    setProgressPercent(80)
     try {
+      if (isIncludingManifest) {
+        const manifestResponse = await createManifestFromDownloadListV2(accessToken)
```

Comment:
Even for a very large list, this is relatively fast.  Each package takes much longer to collect the files (open this immediately since the presigned URL may expire by the time the first package is ready!)